frontend/src/main.tsx

```diff
@@ -0,0 +1,22 @@
+import ReactDOM from 'react-dom/client';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom'; // Correct imports
+import Layout from './Layout';
+import { Assignments, Grading, Settings, Dashboard} from './pages';
+import "./index.css"
+
+function App(): JSX.Element {
+  return (
+    <Router basename="/vite-template"> 
```

Comment:
Update basename unless we are holding as placeholder until we start actual development instead of boiler plate